### ISSUE TITLE
Question regarding the test set

### ISSUE THREAD

**Issue by dain5832** [tsinghua-fib-lab/DICE#6]
Hi!
I was trying to run experiments using the data you have uploaded.
However, it seems like there's only one test_coo_record.npz. It is the non IID testset?
Also, for the train set I would like to clarify if train_blend_coo_adj_graph.npz represents the one used for dice.

Thanks for your help!

**Comment by DavyMorgan** [comment]
Yes, we only evaluate for the non-IID scenario (test_coo_record.npz), and you are right that train_blend_coo_adj_graph.npz represents the one used for dice.

**Comment by dain5832** [comment]
Thanks! Also, if I want to run the experiment on custom data which is only consisted of train and test set, do I need to create a skewed train set? I'm asking this because it seems like the dice dataloader uses both the train and skewed train set. or is it just okay to comment all lines that includes skewed part.

**Comment by DavyMorgan** [comment]
I think it is okay to only use the normal train set without a skewed train set.
